### ISSUE TITLE
[REVIEW]: BrkRaw: A comprehensive tool for accessing raw Bruker Biospin MRI data

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2277]
**Submitting author:** @dvm-shlee (<a href="http://orcid.org/0000-0001-5292-0747">Sung-Ho Lee</a>)
**Repository:** <a href="https://github.com/brkraw/bruker" target ="_blank">https://github.com/brkraw/bruker</a>
**Version:** 0.3.3
**Editor:** @richardjgowers
**Reviewer:** @mih, @mwegrzyn, @PeerHerholz
**Archive:** Pending

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/0a09f27cf34a1839f0dd71436fdf2838/status.svg)](https://joss.theoj.org/papers/0a09f27cf34a1839f0dd71436fdf2838)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/0a09f27cf34a1839f0dd71436fdf2838"><img src="https://joss.theoj.org/papers/0a09f27cf34a1839f0dd71436fdf2838/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/0a09f27cf34a1839f0dd71436fdf2838/status.svg)](https://joss.theoj.org/papers/0a09f27cf34a1839f0dd71436fdf2838)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/brkraw/bruker" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@mih & @mwegrzyn & @PeerHerholz, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @richardjgowers know.

✨ **Please try and complete your review in the next six weeks** ✨ 

### Conflict of interest

- [ ] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [ ] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/brkraw/bruker">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Contribution and authorship:** Has the submitting author (@dvm-shlee) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/brkraw/bruker">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@dvm-shlee) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/brkraw/bruker">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@dvm-shlee) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @mih, @mwegrzyn, @PeerHerholz it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.21105/joss.00354 is OK
- 10.3389/fninf.2013.00045 is OK
- 10.1038/sdata.2016.44 is OK
- 10.5281/zenodo.3745545 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02277/joss.02277/10.21105.joss.02277.pdf)

**Comment by richardjgowers** [comment]
@mih & @mwegrzyn & @PeerHerholz just checking in on how this review is going.  If you have any questions let me know

**Comment by mih** [comment]
Hey @richardjgowers, intense online conference going on. I need a bit longer.

**Comment by PeerHerholz** [comment]
Hi @richardjgowers, same as @mih (actually the same conferences, hehe). I'll try my best to get to it over the weekend. Sorry for the delay!

**Comment by mwegrzyn** [comment]
Dear @richardjgowers,
thank you for your message. I will complete the review within the deadline.
Best regards,
Martin Wegrzyn

**Comment by PeerHerholz** [comment]
Hi @richardjgowers,

I just did the first round of my review. I want to provide precise feedback to the authors regarding the points I did not check and thus need to be addressed in my opinion. This should be done here, correct?

**Comment by richardjgowers** [comment]
Sure or raise them as issues on the repo if that’s more appropriate.

**Comment by PeerHerholz** [comment]
A opened an issue outlining my comments [here](https://github.com/BrkRaw/bruker/issues/13).

**Comment by mih** [comment]
Apparently, I failed to accept the invitation for this repo in time. Therefore I cannot edit the checklist at the moment. Is there a possibility to resend the invitation?

**Comment by danielskatz** [comment]
@whedon re-invite @mih as reviewer

**Comment by whedon** [comment]
OK, the reviewer has been re-invited.

@mih please accept the invite by clicking this link: https://github.com/openjournals/joss-reviews/invitations

**Comment by mwegrzyn** [comment]
Dear colleagues,

my review of BrkRaw can be found here: https://github.com/BrkRaw/bruker/issues/22

Best regards,
Martin Wegrzyn

**Comment by richardjgowers** [comment]
@mih are you still available to review this?

@dvm-shlee you've got comments from a reviewer on the repo still waiting to be addressed

**Comment by labarba** [comment]
hi 👋 – It looks like we may have lost @mih in this review? @richardjgowers, can this be taken to the finish line with the other two reviewers? It looks like we also wait for the author to report on some mods: how are you getting along, @dvm-shlee?

**Comment by richardjgowers** [comment]
@labarba yeah I'm happy on using 2 reviewers here.  I was waiting on reviewer comments here: https://github.com/BrkRaw/bruker/issues/22 to get addressed by the authors (@dvm-shlee)

**Comment by richardjgowers** [comment]
@labarba I'm not sure this review is going anywhere, it looks like review comments made 2 months ago (see previous comment) haven't been acted upon by the author

**Comment by labarba** [comment]
Ping @openjournals/jose-eics  — I have retired from the JOSS board.

**Comment by arfon** [comment]
@dvm-shlee - could you update us on your status here? Are you still interested in pursuing this publication in JOSS?

**Comment by dvm-shlee** [comment]
@arfon Sorry about the lots of delay in revising this, I will update the address of all the commented item soon!

**Comment by arfon** [comment]
@whedon remind @arfon in two weeks

@dvm-shlee - it has now been more than 6 months since we last heard from you. At this point I believe the best next step would be to close this review and encourage a resubmission in the future if you're able to improve the submission.

I'll set a reminder for two weeks to check in again but unless we hear back from you with a firm timeline in the next couple of weeks we will reject this submission.

**Comment by whedon** [comment]
Reminder set for @arfon in two weeks

**Comment by dvm-shlee** [comment]
@arfon Dear editor, I'm really sorry about the long delay. Have some personal issue cannot focus on the coding for a long time. 
And as you suggested, I would like to close this round, and I will re-submit after improving the issue that previous reviewers comments.

Thank you.

**Comment by arfon** [comment]
@whedon withdraw
